File: rei-do-chopp-site/src/app/users/models/responses/user.response.ts
import { RoleResponse } from "../../../roles/models/responses/role.response";

export class UserResponse {
  Id: number;
  FullName: string;
  FirstName: string;
  LastName: string;
  Email: string;
  CreationDate: Date;
  Roles: RoleResponse[];
  Active: boolean;
}

File: rei-do-chopp-api/ReiDoChopp.DataTransfer/Users/Responses/UserResponse.cs
using ReiDoChopp.DataTransfer.Roles;
using ReiDoChopp.DataTransfer.Roles.Responses;

namespace ReiDoChopp.DataTransfer.Users.Responses
{
    public class UserResponse
    {
        public int Id { get; set; }
        public string FullName { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public DateTime CreationDate { get; set; }
        public IList<RoleResponse> Roles { get; set; }
        public bool Active { get; set; }
    }
}
